config/backstop/backstopConfig.js

```diff
@@ -4,7 +4,7 @@
 const basicScenario = {
   label: 'test', // name of the test
   url: 'http://localhost:8080',
-  referenceUrl: '', // put here reference to github-pages with ready project
+  referenceUrl: 'https://mate-academy.github.io/layout_solutions/header/',
```

Comment:
header is too generic. I'd put into under moyo_header. So that we can have more 'headers' in future.